guides/common/modules/ref_dynamic_partition_example.adoc

```diff
@@ -0,0 +1,37 @@
+[[dynamic-partition-example]]
```

Comment:
Please change the ID to match the FCC format: [id="dynamic-partition-example_{context}"]

Reply:
Will do this now. This PR was open for 20 days :) but will raise another one in a few mins.

Reply:
Lol sorry, I did not notice that it is merged already. I was waiting for the issue that you have above to review then.

Reply:
I can easily add a github checker if you want to prevent from lines starting with `[[`.

Reply:
@lzap, this check will fail right away because the biggest part of the docs are not modularized yet. This can be useful but only when everything is modularized, many months from now.